armi/physics/fuelCycle/tests/test_fuelHandlers.py

```diff
@@ -772,6 +772,51 @@ def test_yamlSfpOverridesTrackAssems(self):
         finally:
             os.remove(fname)
 
+    def test_readMovesYaml_loadFromSfp(self):
```

Comment:
I think this test, and the one below it, should use ARMI"s `TempDirectoryChanger`:

https://github.com/terrapower/armi/blob/1dffb1d7cb56cee841ff175b39b09e34255b1026/armi/utils/tests/test_utils.py#L222

It would save you having to do the `finally: os.remove(fname)` at the end, and will be really good at keeping the workspace clean if the test fails in some surprising way.

It also has the benefit of looking the same as all the other tests with the same concerns, so that future devs will know exactly what they are looking at.